lib/sort.js

```diff
@@ -2,12 +2,12 @@
 
 // Compare for array.sort with priority
 //   priority - <string[]>, with priority
+// Returns: <Function>, function(s1, s2)
 //   s1 - <string>, to compare
 //   s2 - <string>, to compare
```

Comment:
You should keep the return type for the returned function:
```suggestion
//   s2 - <string>, to compare
//   Returns: <number>
```